### ISSUE TITLE
Precision without specifying number of digits before and after decimal

### ISSUE THREAD

**Issue by johnnyoshika** [dm4t2/intl-number-input#14]
Thank you for this great library!

It looks like you support 16 digit precision, but the number of digits before and after the decimal needs to be specified ahead of time. Is there a way to support 16 digit precision but allow the decimal to be placed anywhere? For example, can all of these be supported at the same time?

```
1,234,567,890,123,456
123,456,789,012,345.6
12,345,678,901,234.56
1,234,567,890,123.456
123,456,789,012.3456
12,345,678,901.23456
1,234,567,890.123456
```

**Comment by dm4t2** [comment]
No, the precision needs to be configured before. You can use a precision range `{min: 0, max: 6}`, but this would result in max. 9-10 available integer digits: `1000000000.123456`.

Due to accuracy issues, the number expressed by concatenated integer+fraction digits must be less than [Number.MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER):

```
1000000000_123456 < 9007199254740991
 999999999_123456 < 9007199254740991
```

**Comment by johnnyoshika** [comment]
@dm4t2 Thank you
